sound/soc/sof/sof-priv.h

```diff
@@ -131,6 +131,7 @@ struct snd_sof_dsp_ops {
 			     unsigned int core_mask); /* optional */
 	int (*core_power_down)(struct snd_sof_dev *sof_dev,
 			       unsigned int core_mask); /* optional */
+	int (*secondary_core_enable)(struct snd_sof_dev *sof_dev, u32 core_mask);
```

Comment:
this is rather odd. why not just to a core_power_down(sdev, init_core_mask) to disable all but the primary/init core?

Reply:
@plbossart that would leave the secondary cores up and only power down the primary core. That wont be right isnt it?

Reply:
besides as far as TGL is concerned host_managed_cores = init_core_mask. so we're already powering down the primary core. The problem is we dont power off the secondary cores gracefully

Reply:
then do a core_power_down(~init_core_mask) :-)

Reply:
that wont achieve anything on TGL Pierre. these cores are not host managed